atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -172,7 +178,7 @@ public void dropTables(Set<TableReference> tableRefs) {
     public Map<Cell, Value> get(TableReference tableRef, Map<Cell, Long> timestampByCell) {
         //noinspection unused - try-with-resources closes trace
         try (CloseableTrace trace = startLocalTrace("get({}, {} cells)",
-                tableRef, timestampByCell.size())) {
+                LoggingArgs.tableRefValue(tableRef), timestampByCell.size())) {
```

Comment:
see below, but we need to make sure the `LoggingArgs.tableRefValue` code path is allocation free if tracing is not enabled. If tracing is enabled, the message formatter will call `toString()` on these args, so they should be lazily computed